### DISCUSSION TITLE
High latency and bandwidth consumption over wifi.

### DISCUSSION THREAD

**Discussion by SauravDharwadkar** [eProsima/Fast-DDS#3161] [Support]
I'm facing same issues high latency 40-50 ms and even higher sometime, I'm using same settings with 35+ nodes.  Also i noticed that when i start adding new nodes in system the bandwidth or data consumption of existing node is also increased.( Measure using access point monitoring)

_Originally posted by @SauravDharwadkar in https://github.com/eProsima/Fast-DDS/issues/157#issuecomment-1356090547_

**Comment by JLBuenoLopez** [comment]
Thanks @SauravDharwadkar for your contribution.

According to our [CONTRIBUTING guidelines](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md#issues-and-support), the proper forum for specific support cases is the [Support forum](https://github.com/eProsima/Fast-DDS/discussions/categories/support). Also, in order to be able to help you some more information would be needed. You may consider looking at the issue template to know the minimum information required, though the more information you can provide, the better.

Finally, the developer's team does not usually have a lot time to allocate for the support cases, so you might also consider contacting our [commercial support team](https://github.com/eProsima/Fast-DDS#getting-help) for options.
